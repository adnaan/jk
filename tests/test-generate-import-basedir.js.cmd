cd %b && jk generate --stdout test.js
