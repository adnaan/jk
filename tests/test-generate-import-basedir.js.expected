message: imported from a module in node_modules
